backend/pyproject.toml

```diff
@@ -183,4 +183,4 @@ preview = true
n_version == '3.8'", "sphinxcontrib-devhelp==2.0.0; python_version > '3.8'", "sphinxcontrib-htmlhelp==2.0.1; python_version == '3.8'", "sphinxcontrib-htmlhelp==2.1.0; python_version > '3.8'", "sphinxcontrib-jsmath==1.0.1", "sphinxcontrib-qthelp==1.0.3; python_version == '3.8'", "sphinxcontrib-qthelp==2.0.0; python_version > '3.8'", "sphinxcontrib-serializinghtml==1.1.5; python_version == '3.8'", "sphinxcontrib-serializinghtml==2.0.0; python_version > '3.8'", "stdlib-list==0.11.0", "tempstorage==6.0", "toml==0.10.2", "tomli==2.0.2", "towncrier==24.8.0", "transaction==5.0", "trio-websocket==0.11.1", "trio==0.27.0", "trove-classifiers==2025.1.15.22", "twine==5.1.1", "types-PyYAML==6.0.12.10", "types-toml==0.10.8.5", "typing-extensions==4.12.2", "tzdata==2025.1", "Unidecode==1.3.8", "urllib3-secure-extra==0.1.0", "urllib3==2.2.3", "wadllib==2.0.0", "waitress==3.0.2", "watchdog==6.0.0", "wcwidth==0.2.13", "webencodings==0.5.1", "WebOb==1.8.9", "webresource==1.2", "webtest==3.0.2", "wheel==0.45.1", "wmctrl==0.5", "wrapt==1.17.0", "WSGIProxy2==0.5.1", "wsproto==1.2.0", "z3c.caching==3.0", "z3c.checkversions==2.1", "z3c.dependencychecker==2.15", "z3c.form==5.1", "z3c.formwidget.query==2.0.0", "z3c.jbot==2.0", "z3c.objpath==2.0", "z3c.pt==4.4", "z3c.relationfield==2.0.0", "z3c.unconfigure==2.1", "z3c.zcmlhook==2.0", "zc.buildout==4.0", "zc.lockfile==3.0.post1", "zc.recipe.egg==2.0.7", "zc.recipe.testrunner==3.1", "zc.relation==2.0", "ZConfig==4.1", "zdaemon==5.1", "ZEO==6.0.0", "zest.pocompile==2.0.0", "zest.releaser==9.2.0", "zestreleaser.towncrier==1.3.0", "zExceptions==5.0", "zipp==3.20.2", "ZODB3==3.11.0", "ZODB==6.0", "zodbpickle==4.1.1", "zodbupdate==2.0", "zodbverify==1.2.0", "zope.annotation==5.0", "zope.app.locales==5.0", "zope.browser==3.0", "zope.browsermenu==5.0", "zope.browserpage==5.0", "zope.browserresource==5.1", "zope.cachedescriptors==5.0", "zope.component==6.0", "zope.componentvocabulary==2.3.0", "zope.configuration==6.0", "zope.container==6.1", "zope.contentprovider==6.0", "zope.contenttype==5.1", "zope.copy==5.0", "zope.datetime==5.0.0", "zope.deferredimport==5.0", "zope.deprecation==5.0", "zope.dottedname==6.0", "zope.event==5.0", "zope.exceptions==5.2", "zope.filerepresentation==6.0", "zope.globalrequest==2.0", "zope.hookable==7.0", "zope.i18n==5.2", "zope.i18nmessageid==7.0", "zope.interface==7.2", "zope.intid==5.0", "zope.keyreference==6.0", "zope.lifecycleevent==5.0", "zope.location==5.0", "zope.mkzeoinstance==6.0", "zope.pagetemplate==5.1", "zope.processlifetime==3.0", "zope.proxy==6.1", "zope.ptresource==5.0", "zope.publisher==7.1", "zope.pytestlayer==8.2", "zope.ramcache==3.0", "zope.schema==7.0.1", "zope.security==7.3", "zope.sendmail==6.2", "zope.sequencesort==5.0", "zope.site==5.0", "zope.size==5.0", "zope.structuredtext==5.0", "zope.tal==5.0.1", "zope.tales==6.0", "zope.testbrowser==7.0", "zope.testing==5.0.1", "zope.testrunner==6.6", "zope.traversing==5.0", "zope.viewlet==5.0", "Zope2==4.0", "Zope==5.12", "ZopeUndo==6.0"]
```

Comment:
Is it possible to put one per line so that we can see a useful diff?

Reply:
I'm looking into it. The toml library in using should be able to do it, but it did not work as expected.